array info for Triggers (#7144)
segmentio/segment-docs: src/connections/destinations/actions.md

| Global Settings | Define authentication and connection-related information like API and Secret keys.                                                                                                                                                                                                                                                                                                                      |
| Mappings        | Handle the individual calls to the destination. In them, you define what type of call you want to make to the destination, and what Triggers that call. Individual Destination Actions may come enabled with some predefined mappings to handle common events like Screen calls, Identify calls, and Track calls. Mappings have two components that make this possible: **Triggers** and an **Action**. |

> | Triggers        | Enable you to define *when* the corresponding Action fires. As part of a Trigger, you can use condition-based filters to narrow the scope of the Trigger. <br /> <br /> Self-service users can add a maximum of two conditions per Trigger.  <br /> <br /> Triggers do not currently support matching on event fields containing ‘.$’ or ‘.$.’, which reference an array type.

Reviewer: ```suggestion
| Triggers        | Enable you to define *when* the corresponding Action fires. As part of a Trigger, you can use condition-based filters to narrow the scope of the Trigger. <br /> <br /> Self-service users can add a maximum of two conditions per Trigger.  Triggers don't support matching on event fields containing ‘.$’ or ‘.$.’, which reference an array type.
```

>                                                                                                                                                              |

Reviewer: ```suggestion
```
